### ISSUE TITLE
JRE failure when running FEST tests

### ISSUE THREAD

**Issue by jim618** [bitcoin-solutions/multibit-hd#739]
When running FEST tests I see the stack trace:
```
[2015-10-05 12:25:40,053] INFO  [main] o.m.h.u.c.MainController - Starting hardware wallet service: TREZOR 
```
 A fatal error has been detected by the Java Runtime Environment:
```
  SIGSEGV (0xb) at pc=0x00007fff8f8a10db, pid=9676, tid=74243

 JRE version: Java(TM) SE Runtime Environment (7.0_40-b43) (build 1.7.0_40-b43)
 Java VM: Java HotSpot(TM) 64-Bit Server VM (24.0-b56 mixed mode bsd-amd64 compressed oops)
 Problematic frame:
 [2015-10-05 12:25:40,056] INFO  [main] o.m.h.u.c.MainController - Starting hardware wallet service: KEEP_KEY 
[2015-10-05 12:25:40,056] DEBUG [safe-scheduled-monitor-hw-client-0] o.m.h.h.t.c.TrezorHardwareWalletClient - Attempting to connect... 
[2015-10-05 12:25:40,056] DEBUG [safe-scheduled-monitor-hw-client-0] o.m.h.h.t.w.v.TrezorV1HidHardwareWallet - Suspect recently detached device. Attempting to locate... 
[2015-10-05 12:25:40,058] DEBUG [safe-scheduled-monitor-hw-client-0] o.m.h.h.k.c.KeepKeyHardwareWalletClient - Attempting to connect... 
[2015-10-05 12:25:40,058] DEBUG [safe-scheduled-monitor-hw-client-0] o.m.h.h.k.w.v.KeepKeyV1HidHardwareWallet - Suspect recently detached device. Attempting to locate... 
[2015-10-05 12:25:40,068] INFO  [safe-scheduled-monitor-hw-client-0] o.m.h.h.k.w.v.KeepKeyV1HidHardwareWallet - Device not attached 
[2015-10-05 12:25:40,069] DEBUG [safe-scheduled-monitor-hw-client-0] o.m.h.h.k.w.v.KeepKeyV1HidHardwareWallet - Failed to locate. Device must be detached. 
[2015-10-05 12:25:40,072] DEBUG [safe-fixed-message-events-0] o.m.h.h.c.e.MessageEvents - Firing 'message' event type: DEVICE_DETACHED for KEEP_KEY 
[2015-10-05 12:25:40,074] DEBUG [safe-fixed-message-events-0] o.m.h.h.c.f.HardwareWalletContext - Received message event: 'DEVICE_DETACHED' 
[2015-10-05 12:25:40,074] DEBUG [safe-fixed-message-events-0] o.m.h.h.k.c.KeepKeyHardwareWalletClient - Performing client soft detach... 
[2015-10-05 12:25:40,074] DEBUG [safe-fixed-message-events-0] o.m.h.h.c.f.HardwareWalletContext - Reset to 'detached' 
[2015-10-05 12:25:40,075] DEBUG [safe-fixed-message-events-0] o.m.h.h.c.f.HardwareWalletStates - Transitioning to 'detached' state 
[2015-10-05 12:25:40,078] DEBUG [safe-fixed-hardware-wallet-events-0] o.m.h.h.c.e.HardwareWalletEvents - Firing 'hardware wallet' event: SHOW_DEVICE_DETACHED for KEEP_KEY 
[2015-10-05 12:25:40,079] DEBUG [safe-fixed-hardware-wallet-events-0] o.m.h.u.c.MainController - Received hardware event: 'SHOW_DEVICE_DETACHED' 
[2015-10-05 12:25:40,079] DEBUG [safe-fixed-message-events-0] o.m.h.h.c.f.HardwareWalletContext - Discarded message event: 'DEVICE_DETACHED' (different device) 
[2015-10-05 12:25:40,079] DEBUG [safe-fixed-message-events-0] o.m.h.h.c.e.MessageEvents - Completed 'message' event: DEVICE_DETACHED 
[2015-10-05 12:25:40,081] DEBUG [safe-fixed-hardware-wallet-events-0] o.m.h.u.c.MainController - Selected deferred credentials type: PASSWORD 
[2015-10-05 12:25:40,081] DEBUG [safe-fixed-hardware-wallet-events-0] o.m.h.h.c.e.HardwareWalletEvents - Completed 'hardware wallet' event: SHOW_DEVICE_DETACHED 
[2015-10-05 12:25:40,112] INFO  [AWT-EventQueue-0] o.m.h.u.f.t.AbstractFestTest - FEST initialising UI... 
[2015-10-05 12:25:40,112] DEBUG [AWT-EventQueue-0] o.m.h.u.MultiBitHD - Initialising UI... 
[2015-10-05 12:25:40,112] DEBUG [AWT-EventQueue-0] o.m.h.u.MultiBitHD - Starting the clock for hardware wallet initialisation 

C  [IOKit+0x260db]  IOHIDDeviceScheduleWithRunLoop+0x26

 Failed to write core dump. Core dumps have been disabled. To enable core dumping, try "ulimit -c unlimited" before starting Java again

[2015-10-05 12:25:40,272] DEBUG [AWT-EventQueue-0] o.m.h.u.p.b.m.MacApplicationBuilder - Found EAWT libraries so building native bridge 
[2015-10-05 12:25:40,289] DEBUG [AWT-EventQueue-0] o.m.c.c.SafeExecutors - New fixed thread pool with 1 threads: 'play-received' 
[2015-10-05 12:25:40,291] DEBUG [AWT-EventQueue-0] o.m.h.u.MultiBitHD - Loading Nimbus LaF... 
 An error report file with more information is saved as:
 /sers/im/deaProjects/ultibit-hd/s_err_pid9676.log

 If you would like to submit a bug report, please visit:
   http:/bugreport.sun.com/ugreport/rash.jsp
 The crash happened outside the Java Virtual Machine in native code.
 See problematic frame for where to report the bug.


Process finished with exit code 134
```

edit: removed # as was being interpreted  as markdown

**Comment by jim618** [comment]
Assigned to Gary as it is most likely caused by something in the 'Trezor/ KeepKey plugging in and out' code.

**Comment by jim618** [comment]
In the most recent run of the FEST tests I didn't see any instances of these, not the MBHD CrashReporter whilst tests were running so closing.
